### ISSUE TITLE
fp-waiting loading animation only visible on hover

### ISSUE THREAD

**Issue by blacktrash** [flowplayer/flowplayer#1218]
The loading animation should be visible independently of whether the user hovers over the player area or not.
As this can be fixed by:
```css
.flowplayer.is-loading .fp-waiting, .flowplayer.is-seeking .fp-waiting {
  display: block !important; /* !important flag added
}
```
this looks a lot like a simple oversight.

See: https://flowplayer.com/forum/#!/skinning_css:player-doesnt-show-waiting

**Comment by bbbo** [comment]
@nnarhinen this doesn't seem to be fixed (or a regression) in 7.2.6.:

http://www.manicmovies.com/watch.php?id=347#.WxVuNJ_7SiN

shows loading animation only when hovering

**Comment by nnarhinen** [comment]
Please don't re-open issues. It messes up the milestoning.
